RoaringBitmap/src/test/java/org/roaringbitmap/longlong/TestRoaring64Bitmap.java

```diff
@@ -25,11 +25,14 @@
 import java.util.List;
 import java.util.Random;
 import java.util.Set;
+
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 import org.roaringbitmap.RoaringBitmap;
 import org.roaringbitmap.Util;
 
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
please don't import the star

Reply:
I mean don't fold the imports as a `org.junit.jupiter.api.Assertions.*`

Reply:
ok，